build.sbt

```diff
@@ -1,5 +1,4 @@
-ThisBuild / tlBaseVersion := "0.23"
-ThisBuild / tlMimaPreviousVersions ++= (0 to 11).map(y => s"0.23.$y").toSet
+ThisBuild / tlBaseVersion := "0.24"
```

Comment:
I couldn't decide whether or not to bump the base version. We can make this change binary-compatibly, it's more about semantics.